Integrations/Panorama/Panorama.py

```diff
@@ -1792,45 +1792,50 @@ def panorama_get_url_category(url):
     return s.split(' ')[1]
 
 
-def populate_url_filter_category_from_context(url_category_hr):
-    url_filter_category = demisto.dt(demisto.context(),
-                                     'Panorama.URLFilter(val.Category === "{0}")'.format(url_category_hr['Category']))
-
+def populate_url_filter_category_from_context(category):
+    url_filter_category = demisto.dt(demisto.context(), f'Panorama.URLFilter(val.Category === "{category}")')
     if not url_filter_category:
-        url_filter_category = {
-            'Category': url_category_hr['Category'],
-            'URL': []
-        }
-
-    if type(url_filter_category) is dict:
-        url_filter_category = [url_filter_category]
-
-    url_filter_category[0]['URL'] += [url_category_hr['URL']]
-
-    return url_filter_category
+        return []
+
+    if type(url_filter_category) is list:
+        return url_filter_category[0].get("URL")
+    else:  # url_filter_category is a dict
+        context_urls = url_filter_category.get("URL", None)  # pylint: disable=no-member
+        if type(context_urls) is str:
+            return [context_urls]
+        else:
+            return context_urls
 
 
 def panorama_get_url_category_command():
     """
     Get the url category from Palo Alto URL Filtering
     """
-    url = demisto.args()['url']
+    urls = argToList(demisto.args()['url'])
 
-    category = panorama_get_url_category(url)
-
-    url_category_hr = {
-        'URL': url,
-        'Category': category
-    }
-
-    url_category_output = populate_url_filter_category_from_context(url_category_hr)
+    categories_dict: Dict[str, list] = {}
```

Comment:
did you test this wit a single url?

Reply:
yes

Reply:
yes, it's working
<img width="734" alt="Screen Shot 2019-07-31 at 12 03 17" src="https://user-images.githubusercontent.com/37335599/62198919-360a9500-b38b-11e9-984a-ce780087a437.png">